packages/reactive_ble_mobile/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 4.0.1
+
+ * Add support for iOS 15
```

Comment:
The first character at this line is space. It's not needed for a top-level list item in Markdown. Also, compare with older items.

Reply:
you were quick indeed I fixed it now :)